docs/changelog.md

```diff
@@ -1,5 +1,5 @@
-=========
-Changelog
-=========
+# Changelog
 
+```{eval-rst}
 .. include:: ../CHANGELOG.rst
```

Comment:
Might be as easy update the CHANGELOG to markdown now. It'll make it easier to work with in GitHub.

What I did for msprime is put in a soft link to the CHANGELOG.md file in the project root here. Works fine.